src/components/clusterConfiguration/discoveryImageModal.tsx

```diff
@@ -50,7 +50,9 @@ export const DiscoveryImageModalButton: React.FC<DiscoveryImageModalButtonProps>
 
 const validationSchema = Yup.object().shape({
   proxyUrl: Yup.string().url('Provide a valid URL.'),
-  sshPublicKey: sshPublicKeyValidationSchema,
+  sshPublicKey: sshPublicKeyValidationSchema.required(
+    'SSH key is required for debugging the host registration.',
```

Comment:
I'm in mixed feelings about this, I dont know if we want to force users, but tell them that they would need to generate a new iso if it fails.... so on one hand you are making everyone life harder just in case it would fail...?

Reply:
What about keeping it for M1 for troubleshooting and remove it later?